reference/toolstacks/php/ezplatform/README.md

```diff
@@ -0,0 +1,35 @@
+# Getting started
+
+## Prerequisites
```

Comment:
What does `Prerequisites` mean? ("Things to install locally in order to build locally"? not that important IMO)